test/acceptance/accessibility.py

```diff
@@ -24,16 +24,11 @@ def _configure_a11y_audit_config(self, page):
         page.a11y_audit.config.set_scope(
             exclude=[
                 "#footer-edx-v3",  # Links to Facebook, Twitter, etc. Populated in production, but not here.
-                ".instructor-info-action",  # Staff Debug Info link (general for all XBlocks).
+                ".instructor-info-action",  # Staff Debug Info link (general for all XBlocks)
+                ".is--disabled", # TODO: TNL-5183
```

Comment:
Buttons that are disabled only via a class still fail color contrast. We can remove this exclude when tackling TNL-5183 (disabled buttons should be truly disabled...).